crucible/src/Lang/Crucible/FunctionHandle.hs

```diff
@@ -231,3 +232,7 @@ searchHandleMap nm fnTyRepr (FnHandleMap m) =
       case testEquality (handleType h) fnTyRepr of
         Just Refl -> Just (h,x)
         Nothing -> Nothing
+
+handleMapToHandles :: FnHandleMap f -> [SomeHandle]
+handleMapToHandles (FnHandleMap m) =
+  map (\(Some (HandleElt handle _)) -> SomeHandle handle) (MapF.elems m)
```

Comment:
@robdockins Is this alright? Would this break an important abstraction barrier?

I'm using this [here](https://github.com/GaloisInc/crucible/pull/703/files#diff-e76f47d5b800a8b1d75a4bed25c18ba59ed5181cc7b4538079604b32c7f063f1R89-R97), the point is that I want to define no-op overrides for functions that have declarations but not definitions, but I need to *not* do that for functions that already have corresponding CFGs, like `free`.

Reply:
It doesn't look like the `FnHandleMap` maintains any invariants